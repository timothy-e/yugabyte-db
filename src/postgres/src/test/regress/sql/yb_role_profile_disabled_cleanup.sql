ALTER USER disabled_user PROFILE DETACH;
DROP USER disabled_user;
DROP PROFILE disabled_profile;