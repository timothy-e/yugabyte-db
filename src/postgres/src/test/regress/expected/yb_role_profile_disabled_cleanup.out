DROP USER disabled_user;
DROP PROFILE disabled_profile;
